pkg/cli/gptscript.go

```diff
@@ -84,7 +84,7 @@ func New() *cobra.Command {
 		&Parse{},
 		&Fmt{},
 		&SDKServer{
-			GPTScript: root,
+			GPTScript: *root,
```

Comment:
Is this change required?

Reply:
required. part of the problem is that our cli builder package doesn't properly detect the option field when it's a pointer.